cockatrice/src/tab_deck_editor.cpp

```diff
@@ -712,6 +719,18 @@ bool TabDeckEditor::actSaveDeck()
     return false;
 }
 
+bool TabDeckEditor::actExportDeck()
+{
+    DeckLoader *const deck = deckModel->getDeckList();
+    QString decklistUrl = deck->exportDeckToDecklist();
+    QMessageBox msgBox(this);
+    msgBox.setWindowTitle("Export to Decklist.org");
+    msgBox.setTextFormat(Qt::RichText);   //this is what makes the links clickable
+    msgBox.setText("<a href='"+QUrl(decklistUrl).toEncoded()+"'>Click here to view your deck on Decklist.org</a>");
+    msgBox.exec();
+    return true;
```

Comment:
Instead of a message box, you can look into how tappedout/deckstat code auto-opens the URL in the background. Would be more effective and consistent

Reply:
Message box removed in latest commit, instead I'm just opening the link in a browser.